src/core/Geometry.js

```diff
@@ -1043,6 +1043,51 @@ THREE.Geometry.prototype = {
 
 		}
 
+		var bones = [];
+		var skinIndices = [];
+		var skinWeights = [];
+		var influencesPerVertex = 0;
+
+		if ( this.bones !== undefined && this.skinIndices !== undefined && this.skinWeights !== undefined ) {
```

Comment:
Right. I've just now noticed that there're "this.skinWeights = []; this.skinIndices = [];" in Geometry constructor.
I should check only if this.bones !== undefined.
(But why no "this.bones = [];" in the constructor?)